fontforgeexe/cvimportdlg.c

```diff
@@ -456,48 +582,60 @@ return( true );
+		ShowImportOptions(ip, d->opts_shown, sp_scale);
+		d->done = FVImportImageTemplate((FontViewBase *) d->fv,temp,format,toback,true,ip);
+	    } else if ( format==fv_eps ) {
+		ShowImportOptions(ip, d->opts_shown, sp_eps);
+		d->done = FVImportImages((FontViewBase *) d->fv,temp,format,toback,true,ip);
+	    } else if ( format==fv_epstemplate ) {
+		ShowImportOptions(ip, d->opts_shown, sp_eps);
+		d->done = FVImportImageTemplate((FontViewBase *) d->fv,temp,format,toback,true,ip);
+	    } else if ( format==fv_pdf ) {
+		ShowImportOptions(ip, d->opts_shown, sp_eps);
+		d->done = FVImportImages((FontViewBase *) d->fv,temp,format,toback,true,ip);
+	    } else if ( format==fv_pdftemplate ) {
+		ShowImportOptions(ip, d->opts_shown, sp_eps);
+		d->done = FVImportImageTemplate((FontViewBase *) d->fv,temp,format,toback,true,ip);
+	    } else if ( format==fv_svg ) {
+		ShowImportOptions(ip, d->opts_shown, sp_svg);
+		d->done = FVImportImages((FontViewBase *) d->fv,temp,format,toback,true,ip);
+	    } else if ( format==fv_svgtemplate ) {
+		ShowImportOptions(ip, d->opts_shown, sp_svg);
+		d->done = FVImportImageTemplate((FontViewBase *) d->fv,temp,format,toback,true,ip);
+	    } else if ( format==fv_glif )
+		d->done = FVImportImages((FontViewBase *) d->fv,temp,format,toback,true,ip);
 	    else if ( format==fv_gliftemplate )
-		d->done = FVImportImageTemplate((FontViewBase *) d->fv,temp,format,toback,-1);
+		d->done = FVImportImageTemplate((FontViewBase *) d->fv,temp,format,toback,true,ip);
 	    else if ( format>=fv_pythonbase )
-		d->done = FVImportImages((FontViewBase *) d->fv,temp,format,toback,-1);
+		d->done = FVImportImages((FontViewBase *) d->fv,temp,format,toback,true,ip);
 	} else if ( d->bv!=NULL )
 	    d->done = BVImportImage(d->bv,temp);
 	else {
 	    d->done = true;
-	    if ( format==fv_image )
-		ImportImage(d->cv,temp);
-	    else if ( format==fv_eps )
-		ImportPS(d->cv,temp);
-	    else if ( format==fv_pdf )
-		ImportPDF(d->cv,temp);
-	    else if ( format==fv_plate )
-		ImportPlate(d->cv,temp);
-	    else if ( format==fv_svg )
-		ImportSVG(d->cv,temp);
-	    else if ( format==fv_glif )
-		ImportGlif(d->cv,temp);
-	    else if ( format==fv_fig )
-		ImportFig(d->cv,temp);
+	    if ( format==fv_image ) {
+		ShowImportOptions(ip, d->opts_shown, sp_scale);
+		ImportImage(d->cv,temp,ip);
+	    } else if ( format==fv_eps ) {
+		ShowImportOptions(ip, d->opts_shown, sp_eps);
+		ImportPS(d->cv,temp,ip);
+	    } else if ( format==fv_pdf ) {
+		ShowImportOptions(ip, d->opts_shown, sp_eps);
+		ImportPDF(d->cv,temp,ip);
+	    } else if ( format==fv_plate )
+		ImportPlate(d->cv,temp,ip);
+	    else if ( format==fv_svg ) {
+		ShowImportOptions(ip, d->opts_shown, sp_svg);
+		ImportSVG(d->cv,temp,ip);
+	    } else if ( format==fv_glif )
+		ImportGlif(d->cv,temp,ip);
+	    else if ( format==fv_fig ) {
+		ImportFig(d->cv,temp,ip);
 #ifndef _NO_PYTHON
-	    else if ( format>=fv_pythonbase )
+	    } else if ( format>=fv_pythonbase )
```

Comment:
This bracket messes up compiling with python disabled.

Reply:
should be fixed.